Homework6

strwatcher opened the pull request.

strwatcher committed:
Add location permission checker

strwatcher committed:
Add map and user location tracking

strwatcher committed:
Fix distanceTo finction

strwatcher committed:
Add service in manifest
Remove RequiresSDK redundancies

strwatcher committed:
tracking, drawing path line, updating distance in db and in view

strwatcher committed:
Add finish activity button functionality
Save finish time in db now

strwatcher committed:
Add all functionality that was needed

Now user can start only one activity and after that the start button is switch to continue button

(about a million different bug-fixes)

blnvfeip reviewed (commented):
У вас норм проект собирается? (если склонировать с гитхаба в чистую директорию, не из той в которой вы работаете)
Ошибку вываливает `Schema '1.json' required for migration was not found at the schema out folder`

strwatcher commented:
Все нормально собирается да, у меня тоже была эта ошибка, я ее фиксил как-то. Сейчас гляну как. Может не добавил что-то в коммит

strwatcher commented:
> У вас норм проект собирается? (если склонировать с гитхаба в чистую директорию, не из той в которой вы работаете) Ошибку вываливает `Schema '1.json' required for migration was not found at the schema out folder`

А, понял в чем проблема

strwatcher committed:
remove Migrations

strwatcher commented:
Удалил миграции в общем, уверен, что можно было нормально решить, но насколько я понял надо было изначально в грэдле прописать путь для экспорта схем, потому что рум почему-то автоматичекски этого не делает(а может я и не прав)
Энивэй сейчас потестил в чистой директории и все работает

blnvfeip reviewed (commented):
– продолжительность не выводится на экране детализации
– правильней будет на экран детализации передавать ид активности, а не весь объект сразу, и уже во фрагменте идти в БД и запрашивать активность по этому ид

strwatcher committed:
duration not showed issue fixed

now only id is translated to next fragment in MyActivities -> MyActivityInfo direction

strwatcher committed:
Merge remote-tracking branch 'origin/Homework6' into Homework6

strwatcher committed:
duration now show and in the user activity info

strwatcher commented:
Поправил
